README.md

```diff
@@ -91,6 +96,9 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+If you are encountering `undefined method ‘attribute’ for ActiveModel::Serializer class` error, you might be using version `0.9.x`.
+Please see [How to Alias Attribute in 0.9.x](#/tree/0-9-stable#alias-attribute)
```

Comment:
I like having a troubleshooting section. (though agree this doesn't belong in the 0.10 documentation)